.github/workflows/app_engine.yml

```diff
@@ -19,10 +35,40 @@ jobs:
       run: |
         python -m pip install --upgrade pip
         pip install -r gcp/app-engine/requirements.txt
-    - name: Test with pytest
+        sudo apt-get update
+        sudo apt-get install -y sqitch
+    - name: Wait for Postgres to be ready
+      run: |
+        until pg_isready -h localhost -p 5432; do
+          echo "Waiting for Postgres to be ready..."
+          sleep 2
+        done
+    - name: Apply Sqitch migrations
+      working-directory: gcp/migrations
+      env:
+        PGUSER: postgres
+        PGPASSWORD: password
+        PGHOST: localhost
```

Comment:
from what i've read, using "postgres" here should work given the docker service name above, but the test was failing to resolve the host. Localhost seems to work though